src/AppInstallerRepositoryCore/Microsoft/Schema/1_0/Interface.h

```diff
@@ -36,6 +36,10 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_0
         // Version 1.2
         Utility::NormalizedName NormalizeName(std::string_view name, std::string_view publisher) const override;
 
+        // Validate the dependencies of the given manifest.
+        bool ValidateManifest(SQLite::Connection& connection, const Manifest::Manifest& manifest) const override;

```

Comment:
I don't like this design of putting the validation inside the ISQLiteIndex.  It does not fit with the rest of the design; the index is a data store and the only reason to put the validation code inside would be that the data was potentially going to change so much that it was no longer convertible.

It also means that this is less flexible.  For instance, if we wanted to keep this table and use it during `install`, we would have to have new methods to actually extract the dependency information.